backend/src/main/java/ai/verta/modeldb/experimentRun/subtypes/PredicatesHandler.java

```diff
@@ -180,15 +170,12 @@ private String getEntityName() {
   }
 
   private InternalFuture<QueryFilterContext> getContextFilterForSingleStringFields(
-      KeyValueQuery predicate, String key, Value value, String bindingName) {
+      KeyValueQuery predicate, String key, Object value, String bindingName) {
```

Comment:
I really, really don't like this. Can we do something better with concrete types?

Reply:
Fixed